packages/calling/src/CallingClient/constants.ts

```diff
@@ -12,6 +12,8 @@ export const DEFAULT_LOCAL_CALL_ID = 'DefaultLocalId';
 export const DEFAULT_REHOMING_INTERVAL_MAX = 120;
 export const DEFAULT_REHOMING_INTERVAL_MIN = 60;
 export const DEFAULT_SESSION_TIMER = 1000 * 60 * 10;
+export const DEFAULT_CALL_STATE_TIMER = 20 * 1000;
```

Comment:
How did we arrive at these times.. Does it seem really high? Could we use timers in the meeting plugin to get a better idea? Or perhaps check with calling team on the usual behavior.

Reply:
Discussed with Krishna.
Every state change should be instantaneous so a 15 seconds timeout is enough
For Ringing/Alerting -> Connected, we can have a longer timeout leaving 60 seconds.

Reply:
> For Ringing/Alerting -> Connected, we can have a longer timeout leaving 60 seconds.

The code changes seem to suggest that we are waiting for 60 seconds between Connect > Established?